packages/combobox/src/useCombobox.ts

```diff
@@ -449,11 +453,11 @@ export const useCombobox = <
       };
 
       if (isEditable && triggerContainsInput) {
-        const handleClick = (event: MouseEvent) => {
+        const handleClick = (event: React.MouseEvent) => {
           if (disabled) {
             event.preventDefault();
           } else if (isAutocomplete) {
-            triggerProps.onClick(event);
+            triggerProps.onClick && triggerProps.onClick(event);
```

Comment:
I don't think it does. Instead of testing for invalid `triggerProps`, the updated Downshift is not guaranteeing that the trigger props will contain a valid `onClick` ... and we can't `triggerProps.onClick?(event)` :wink: